spec/message_manager_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'message_manager'
+
+describe Message do
+  describe '.post_peep' do
+    it 'saves a peep in a messages table in the database' do
+      peep = Message.post_peep(content: 'My new peep')
+      persisted_data = persisted_data(id: peep.id, table: 'messages')
+
+      expect(peep).to be_a Message
+      expect(peep.id).to eq persisted_data['id']
+      expect(peep.content).to eq 'My new peep'
+    end
+  end
+
+  describe '.show_peeps' do
+    it 'returns a list of peeps' do
+      time_now = Time.parse("2018-11-22 12:00:00")
+      allow(Time).to receive(:now) { time_now }
```

Comment:
This is a robust way of checking the time. Buy hard coding it you are making sure that the test runs the same each time. 

You might want to think about testing multiple different times though so that you catch edge cases. 

For example differences like `09:01` and `9:01` where you have leading zeros etc.